portafly/src/tests/factories/developer-account.ts

```diff
@@ -0,0 +1,14 @@
+import { Factory } from 'fishery'
+import { IDeveloperAccount } from 'types'
+
+const DeveloperAccount = Factory.define<IDeveloperAccount>(({ sequence }) => ({
```

Comment:
Can we have another type of account, a not-developer account ?

Reply:
We eventually will, and when we do, we could add the factory